boards/cubepilot/cubeorange/test.px4board

```diff
@@ -12,3 +12,4 @@ CONFIG_MODULES_TEMPERATURE_COMPENSATION=n
 CONFIG_BOARD_TESTING=y
 CONFIG_DRIVERS_TEST_PPM=y
 CONFIG_SYSTEMCMDS_MICROBENCH=y
+CONFIG_SYSTEMCMDS_BL_UPDATE=n
```

Comment:
@dagar is this ok to save flash for the test platform for the v1.13 backport?

Reply:
Yes that's fine.